experiments/depth-series/depth_session.py

```
@@ -86,10 +86,25 @@ def main():
         load = base.admin(cfg, "load"); print("load:", load, flush=True)
         if load.get("kind") != "state": raise SystemExit(f"the load did not reach a state: {load}")
         if not base.wait_socket(cfg): raise SystemExit("the gate socket never stood")
+        # **The loop that composed this load is checked before a turn is
+        # served through it**, per issue #426: the config's `loop_sha256`
+        # against the load event's composer digest, the shared helper doing
+        # the comparison. A refusal serves no turn, lands typed in the
+        # report, and exits nonzero, so the deposit is a refusal and not a
+        # short session.
+        refused = base.assert_loop(cfg, trace, before)
+        if refused:
+            report["loop_refused"] = refused; run_id = refused["run"]; report["run"] = run_id
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)